bookkeeper-server/src/main/java/org/apache/bookkeeper/client/BookKeeper.java

```diff
@@ -82,7 +89,7 @@
  * <p>The exceptions resulting from synchronous calls and error code resulting from
  * asynchronous calls can be found in the class {@link BKException}.
  */
-public class BookKeeper implements AutoCloseable {
+public class BookKeeper implements AutoCloseable, org.apache.bookkeeper.client.api.BookKeeper {
```

Comment:
not sure if it is standard convention to have same interface and class name - 'Bookkeeper', it would be confusing

Reply:
We have a brand new package o.a.bookkeeper.client.api and ideally new applications will use only this package and nothing else.
I will create a "Builder" for the main BookKeeper client in o.a.bookkeeper.client.api, this way new applications never use explicitly the old package, except from the BKException class

Reply:
@reddycharan there will be a period the old api and new api will co-exist. after we deprecate the old api, the `client.BookKeeper` class will become an impl class only, when we can rename it to BookKeeperImpl.